.github/workflows/swift-toolchain.yml

```diff
@@ -2132,42 +2131,84 @@ jobs:
           - arch: amd64
             cpu: x86_64
             triple: x86_64-unknown-windows-msvc
-
-          # TODO(thebrowsercompany/swift-build#125): Fix the layout of SDK files so we can cross-compile to arm64 
-          # - arch: arm64
-          #   cpu: aarch64
-          #   triple: aarch64-unknown-windows-msvc
+          - arch: arm64
+            cpu: aarch64
+            triple: aarch64-unknown-windows-msvc
 
     steps:
-      - name: Download compilers
+      - name: Download host compilers
         uses: actions/download-artifact@v4
         with:
           name: compilers-amd64
-          path: ${{ github.workspace }}/BuildRoot/Library
+          path: ${{ github.workspace }}/BuildRoot/
 
-      - name: Download devtools
+      - name: Download host devtools
         uses: actions/download-artifact@v4
         with:
           name: devtools-amd64
-          path: ${{ github.workspace }}/BuildRoot/Library
+          path: ${{ github.workspace }}/BuildRoot/
 
-      - name: Download cmark-gfm-amd64
+      - name: Download host Windows runtime
         uses: actions/download-artifact@v4
         with:
-          name: cmark-gfm-amd64-0.29.0.gfm.13
-          path: ${{ github.workspace }}/BuildRoot/Library/cmark-gfm-0.29.0.gfm.13/usr
+          name: windows-sdk-amd64
+          path: ${{ github.workspace }}/BinaryCache/
 
-      - name: Download amd64 Windows Runtime
+      - name: Download host Windows SDK
         uses: actions/download-artifact@v4
         with:
           name: windows-sdk-amd64
-          path: ${{ github.workspace }}/BinaryCache
+          path: ${{ github.workspace }}/BuildRoot/
 
-      - name: Download ${{ matrix.arch }} Windows SDK
+      - name: Update environment variables
+        run: |
+          $SdkRoot = cygpath -w "${{ github.workspace }}/BuildRoot/Developer/SDKs/Windows.sdk"
+          $ToolchainPath = cygpath -w "${{ github.workspace }}/BuildRoot/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin"
+          $RTLPath = cygpath -w "${{ github.workspace }}/BinaryCache/Developer/SDKs/Windows.sdk/usr/bin"
+
+          echo "SDKROOT=${SdkRoot}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          echo "TOOLCHAIN_PATH=${ToolchainPath}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          echo "AR=${ToolchainPath}\llvm-ar.exe" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          echo "${ToolchainPath}" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          echo "${RTLPath}" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+      - name: Patch the SDK to workaround https://github.com/thebrowsercompany/swift-build#50
```

Comment:
@compnerd this isn't the right issue ID. I'm not sure how I lost but I can't find the correct one anymore. Remember where it is?

Reply:
This is somewhat related to thebrowsercompany/swift-build#50 - the issue is that we do not have proper install support in CMake, and the files are being copied to the wrong location.

Reply:
Ah, I see. Thanks for jogging my memory!